guide/contributing/contributing.md

```diff
@@ -1,31 +1,56 @@
 # Contributing to TDS
 
+Role-specific guides and resources:
+
 - [Designer guide](./designer-guide.md)
 - [Developer guide](./developer-guide.md)
 - [Codebase overview](./codebase-overview.md)
 
-## Community
+## Introduction
+
+There are many ways you can contribute to the design system, including submission of bug fixes, documentation improvements,
+and participation in discussions.
+
+We welcome contributions to the currently available components. Check the [component documentation](ref://../components/index.html)
```

Comment:
We've removed `ref` links earlier, should this also be done here?

Reply:
This is still needed when generating staging URLs to the component catalogue. It isn't needed for relative Gitbook page links.